db/migrate/20191104064722_create_restaurants.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRestaurants < ActiveRecord::Migration[5.2]
+  def change
+    create_table :restaurants do |t|
+      t.string :name
+      t.float :long
+      t.float :lat
```

Comment:
I believe it's better if we just store the address as a string and then call a geocoding gem to automatically convert the string into a long/lat combo